src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```
@@ -13,6 +13,7 @@
     <TieredPGO>true</TieredPGO>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <PublishReadyToRun>true</PublishReadyToRun>
+	<RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
```

Also use [Dynamically Adapting To Application Sizes](https://maoni0.medium.com/dynamically-adapting-to-application-sizes-2d72fcb6f1ea) for GC?
```suggestion
    <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
    <GarbageCollectionAdaptationMode>1</GarbageCollectionAdaptationMode>
```